tools/migrate/migrate.go

```diff
@@ -2,12 +2,12 @@
 package migrate
 
 import (
+	"database/sql"
 	"embed"
 	"io/fs"
-	"net/url"
 
 	"github.com/golang-migrate/migrate/v4"
-	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
```

Comment:
I believe we need pgx, not postgres.

Reply:
I'm not sure that makes any difference as the driver is already initialized but changed it anyways... (+ removed a redundant fn)